scripts/linters/pylint_extensions.py

```diff
@@ -165,6 +176,30 @@ def visit_call(self, node):
                             func_name))
                     num_positional_args_unused -= 1
 
+        for arg in keyword_args:
+            # If there is *args and **kwargs in the function definition skip the
+            # check because we can use keywords arguments in function call even
+            # if **kwargs is present in the function definition. See Example:
+            # Function def -> def func(entity_id, *args, **kwargs):
+            # Function call -> func(entity_id='1', a=1, b=2, c=3)
+            # by parsing calling method we get
+            # keyword_arguments = entity_id, a, b, c. but
+            # from the function definition, we will get keyword_arguments = []
```

Comment:
from -> From

Remove but from the previous line.

Reply:
Done.